core/src/main/java/org/opensearch/sql/expression/function/BuiltinFunctionName.java

```diff
@@ -170,6 +170,7 @@ public enum BuiltinFunctionName {
   ASCII(FunctionName.of("ascii")),
   LOCATE(FunctionName.of("locate")),
   REPLACE(FunctionName.of("replace")),
+  POSITION(FunctionName.of("position")),
```

Comment:
can you re-order this list alphabetically please?  (just the Text functions)

Reply:
Addressed in 9a6b7b5d8962cff8af410638da5151e9ee6a36b9